expression/aggregation/base_func.go

```diff
@@ -287,7 +287,8 @@ func (a *baseFuncDesc) typeInfer4BitFuncs(ctx sessionctx.Context) {
 	a.RetTp.SetFlen(21)
 	types.SetBinChsClnFlag(a.RetTp)
 	a.RetTp.AddFlag(mysql.UnsignedFlag | mysql.NotNullFlag)
-	// TODO: a.Args[0] = expression.WrapWithCastAsInt(ctx, a.Args[0])
+	// make cast to int explicit
```

Comment:
Normally we call this an "implicit cast". "Explicit" doesn't fit this case.

Reply:
Maybe we should just remove this comment as the code below pretty much tells for itself.